env

```diff
@@ -1 +1,2 @@
-GATSBY_ENVIRONMENT_BRANCH=develop 
\ No newline at end of file
+GATSBY_ENVIRONMENT_BRANCH=develop
+PORT=8000
```

Comment:
[?] What is this file used for?

Reply:
The docs instruct you to copy this to `.env.development` to set up your environment. I thought it made sense to piggy back on that for the docker env stuff as well.

Reply:
I would rename this file to be `env.development.example` or something like that.